scripts/docker-compose-mysql/README.md

```diff
@@ -0,0 +1,26 @@
+# Cromwell server on MySQL Database
+
+Uses docker-compose to link together a cromwell docker image (built locally with `sbt docker` or available on [dockerhub](https://hub.docker.com/r/broadinstitute/cromwell/)) with a MySQL docker image.
+To change the version of cromwell used, change the tag in `compose/cromwell/Dockerfile`
+
+## Local
+
+`docker-compose up` from this directory will start a cromwell server running on a mysql instance with local backend.
+The default configuration file used can be found at `compose/cromwell/app-config/application.conf`
+To override it, simply mount a volume containing your custom `application.conf` to `/app-config` (see the Jes section below)
```

Comment:
This says to see the jes section but I don't see what in there is providing extra info